src/itemTracker/dungeonTracker.js

```diff
@@ -5,14 +5,21 @@ import Col from "react-bootstrap/Col"
 import dungeonLayout from '../assets/dungeonLayout.png'
 import AreaCounters from '../locationTracker/AreaCounters'
 
+import SV_Entrance from './items/Entrances/SV_Entrance';
+import ET_Entrance from './items/Entrances/ET_Entrance';
+import LMF_Entrance from './items/Entrances/LMF_Entrance';
+import AC_Entrance from './items/Entrances/AC_Entrance';
+import SSH_Entrance from './items/Entrances/SSH_Entrance';
+import FS_Entrance from './items/Entrances/FS_Entrance';
+import SK_Entrance from './items/Entrances/SK_Entrance';
```

Comment:
Consider merging these into a single component (see DungeonName.js)

Reply:
Also I have no idea if I did this during the merge or not, but these imports should be CamelCased, and the file names should be the same

Reply:
Also pls move them into the dungeons folder, and lowercase the E in Entrances (for consistency)